src/components/pages/group/edit/homepage-customization/gallery-setup/gallery-setup-form/index.tsx

```diff
@@ -21,12 +23,16 @@ interface IGallerySetupForm {
   authorName?: string;
   profilePic?: string;
   options?: any[];
+  truncated?: boolean;
 }
 
 export default function GallerySetupFrom({ setIsCreate, galleryList, setGalleryList }) {
   const { t } = useTranslation();
   const [imagePicker, setImagePicker] = useState<boolean>(true);
-  const [defaultValues, setDefaultValues] = useState<IGallerySetupForm | any>();
+  const { currentGroup } = useGlobalState();
+  const [defaultValues, setDefaultValues] = !currentGroup.id
+    ? useState<IGallerySetupForm | any>({ truncated: true })
+    : useState<IGallerySetupForm | any>();
```

Comment:
Instead of this you can use it like below

```jsx
  const [defaultValues, setDefaultValues] = useState<IGallerySetupForm | any>(currentGroup.id ? undefined : { truncated: true })
```